test/jit/test_freezing.py

```diff
@@ -1827,6 +1829,34 @@ def forward(self, x):
                 FileCheck().check("aten::cudnn_convolution_relu").run(frozen_mod.graph)
 
             self.assertEqual(mod_eager(inp), frozen_mod(inp))
+
+    @unittest.skipIf(not torch._C.has_mkldnn, "MKL-DNN build is disabled")
+    def test_hardswish_hardsigmoid(self):
+        with set_default_dtype(torch.float):
+            op_map = {
+                'prim::MKLDNNHardSwish' : F.hardswish,
+                'prim::MKLDNNHardSigmoid' : F.hardsigmoid
+            }
+
+            input_sizes = ([0], [1], [3], [1, 3, 8, 8])
+            for (mkldnn_opname, aten_op) in op_map.items():
+                for size in input_sizes:
+                    for inplace in (True, False):
+                        inplace_str = "_" if inplace else ""
+                        inplace_tgt = "%34" if inplace else "%35"
+                        graph_str = f"""graph(%input.1 : Tensor):
+                            %33 : None = prim::Constant()
+                            %34 : Tensor = aten::to_mkldnn(%input.1, %33)
+                            %35 : Tensor = {mkldnn_opname}{inplace_str}(%34)
+                            return ({inplace_tgt})
+                        """
+                        g = parse_ir(graph_str)
+                        m = self.createFunctionFromGraph(g)
+                        x = torch.rand(size)
+                        x_copy = x.detach().clone()
```

Comment:
nit: why are you detaching and cloning x_copy ?

Reply:
for in-place tests, the reference implementations modify the input in `aten_op(x, inplace=inplace)`. I guess, I could just always use  `F.hardswish(inplace=False)` , might look a bit cleaner.